app/buyers/views/buyers.py

```diff
@@ -92,163 +92,205 @@ def create_new_brief(framework_slug, lot_slug):
         errors = section.get_error_messages(e.message)
 
         return render_template(
-            "buyers/edit_brief_section.html",
-            framework=framework,
+            "buyers/create_brief_question.html",
             data=update_data,
+            brief={},
             section=section,
+            question=section.questions[0],
             errors=errors
         ), 400
 
     return redirect(
-        url_for(".edit_brief_submission",
+        url_for(".view_brief_overview",
                 framework_slug=framework_slug,
                 lot_slug=lot_slug,
-                brief_id=brief['id'],
-                section_id=content.get_next_editable_section_id(section.slug)))
+                brief_id=brief['id']))
 
 
-@buyers.route(
-    '/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/edit/<section_id>',
-    methods=['GET'])
-def edit_brief_submission(framework_slug, lot_slug, brief_id, section_id):
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>', methods=['GET'])
+def view_brief_overview(framework_slug, lot_slug, brief_id):
+    get_framework_and_lot(framework_slug, lot_slug, data_api_client, must_allow_brief=True)
+    brief = data_api_client.get_brief(brief_id)["briefs"]
 
-    framework, lot = get_framework_and_lot(framework_slug, lot_slug, data_api_client,
-                                           status='live', must_allow_brief=True)
+    if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id):
+        abort(404)
+
+    content = content_loader.get_manifest(brief['frameworkSlug'], 'edit_brief').filter({'lot': brief['lotSlug']})
```

Comment:
This ties a bit with the `lot`/`lotSlug` move @TheDoubleK was talking about this morning. Normally, this should be `.filter(brief)`, but it looks like we're trying to move away from `brief['lot']` containing a slug while all questions still use "depends on lot" for lot checks.

We should probably leave this as is right now, but it's worth revisiting later since this limits depends to only work on 'lot', which isn't mentioned anywhere in frameworks repo.